src/test/java/com/education/service/TimetableServiceTest.java

```diff
@@ -70,15 +70,15 @@ public void delete() {
     @Test
     public void update() {
         TimetableBuilder timetableBuilder1 = new TimetableBuilder()
-                .start(LocalTime.of(7,0))
-                .end(LocalTime.of(8,30))
+                .start(LocalTime.of(7, 0))
+                .end(LocalTime.of(8, 30))
                 .date(LocalDate.now())
                 .capacity(5);
         Timetable timetable1 = timetableBuilder1.build();
         timeTableService.save(timetable1);
 
         TimetableBuilder timetableBuilder2 = new TimetableBuilder();
-timetableBuilder2.end(LocalTime.of(8,45));
+        timetableBuilder2.end(LocalTime.of(8, 45));
         Timetable timetable2 = timetableBuilder2.build();
```

Comment:
You can remove "timetableBuilder" variable as described in the previous comment.